### PR TITLE
Removes Hades Chapel from Z2

### PR DESCRIPTION
**Pull request by Core0verload**
This PR removes Hades Chapel from Z2. Having it spawn all the time just for 1 in 200 rounds when Hades actually spawns and gets killed was unwise.

Hades now simply drops his powered staff on death. Said staff is no longer 75-damage bullshit it was, it's damage is now closer to unpowered staff Hades dropped before.

This PR also includes a fix for holodeck areas on z2. If some holodeck modes were broken, this fixes it.

:cl: CoreOverload
del: Hades Chapel is now gone.
/:cl:

### CODE REVIEW

**Comment by MrStonedOne** [comment]
This adds nothing and removes alot

whats the big deal?

ctf does the same, and is rarely used, ayylins do the same thing, and is rarely used, ninja does the same thing, and is rarely used.

like really?

**Comment by Incoming5643** [comment]
the only thing that really bugs me is that there's always a few dozen mobs wandering about every round regardless of if it's being used. If the mobs only spawned in when a pope died it wouldn't be so bad.

File: code/game/gamemodes/miniantags/hades/hades_staff.dm [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+/obj/item/weapon/staff/hades/imbued/attack(mob/living/carbon/human/M, mob/living/carbon/human/user)
+	if(!istype(M))
+		return ..()
+
+	if(world.time > lastJudge + judgeCooldown)
+		var/mob/living/sinPerson = M
+		lastJudge = world.time
+		var/sinPersonchoice = pick("Greed","Gluttony","Pride","Lust","Envy","Sloth","Wrath")
+
+		src.say("Your sin, [sinPerson], is [sinPersonchoice].")
```
Comment (Xhuis) [inline code/game/gamemodes/miniantags/hades/hades_staff.dm]: Wouldn't this make the _staff_ speak out loud instead?

**Comment by xxalpha** [comment]
@Incoming5643 and CTF has like 20 active emitters.

File: code/game/gamemodes/miniantags/hades/hades_staff.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/hades/hades_staff.dm]: also src. is implied

File: code/game/gamemodes/miniantags/hades/hades.dm [hunk 2]
```diff
@@ -129,18 +127,10 @@
...
-					src.visible_message("<span class='warning'><font size=4>[src]'s Staff is flung free as their body explodes.</font></span>")
-					var/obj/structure/ladder/unbreakable/hades/churchLadder = new/obj/structure/ladder/unbreakable/hades(T)
-					var/obj/structure/ladder/unbreakable/hades/bodyLadder = new/obj/structure/ladder/unbreakable/hades(get_turf(src))
-					var/obj/item/weapon/hades_staff/HS = new/obj/item/weapon/hades_staff(get_turf(src))
-					HS.throw_at_fast(pick(orange(src,7)),10,1)
-					churchLadder.up = bodyLadder
-					bodyLadder.down = churchLadder
-					qdel(src)
+
+			src.visible_message("<span class='warning'><font size=4>[src]'s Staff is flung free as their body explodes.</font></span>")
```
Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/hades/hades.dm]: impliiiiied src.

**Comment by Core0verload** [comment]
It's literally old staff code, I just renamed "hades_chapel.dm" to "hades_staff.dm" because hades chapel is no more.

**Comment by Core0verload** [comment]
It's literally old staff code, I just renamed "hades_chapel.dm" to "hades_staff.dm" because hades chapel is no more.

**Comment by Core0verload** [comment]
It's literally old staff code, I just renamed "hades_chapel.dm" to "hades_staff.dm" because hades chapel is no more.

**Comment by sawu-tg** [comment]
@MrStonedOne This is just a grudge PR because I made fun of Core in coderbus, and it's his juvenile way of thinking he's getting back at me.

If you're going to remove it @Core0verload just delete the whole thing instead of literally removing the point of his existence.

**Comment by optimumtact** [comment]
This doesn't make any sense, we're not concerned about space in z2, nor are we concerned about the impact it's going to have on roundstart or during the round.

If we were there are plenty of other targets to aim for first.

**Comment by KorPhaeron** [comment]
Having the jump to mob list full of 30 acolytes is dumb though

**Comment by optimumtact** [comment]
The jump to mob list is full of a bunch of stupid shit anyway, there are plenty of ways to fix that.

**Pull request closed without merging**